content/en/apps/examples/training.md

```diff
@@ -42,19 +42,14 @@ The CHT’s onboarding and training capabilities offer a remote way to provide e
 
 ### Remote Login by App
 
-When creating users, the admin has the option to send a user their credentials via SMS using a [magic link](https://hackernoon.com/magic-links-d680d410f8f7). This generates a new, random and complex password with a 24-hour expiry. If no gateway is set up, the message may be sent via another messaging app. 
+When creating users, the admin has the option to send a user their credentials via SMS using a [magic link](https://hackernoon.com/magic-links-d680d410f8f7). This generates a new, random and complex password with a 24-hour expiry.
 
-![admin-magiclink](admin-magiclink.png)
+By clicking the magic link to log in, the user is able to enter their project's instance directly, bypassing the need to enter their username and password. 
 
-By clicking the magic link to log in, the user is able to enter their project's instance directly, bypassing the need to enter their username and password. If the app is not installed on their phone, it will open in their default browser.
+![magic-link](magic-link.png)
 
-![login.png](login.png)
+{{< see-also page="apps/concepts/access/#remote-login" >}}
```

Comment:
The content here is pretty much a duplicate of what's in this `see-also` link - why not just link to the other page rather than copying a bunch of the content over?

Reply:
Made this more succinct and kept the meat in the Access page